### DISCUSSION TITLE
⭐️Help me get starstruck!⭐️ I will help you too⭐️

### DISCUSSION THREAD

**Discussion by kshitij-halankar** [Schweinepriester/github-profile-achievements#407] [Q&A]
Please give me a star for my repo. I will do the same.

https://github.com/kshitij-halankar/TravelR

Much appreciated. Thanks in advance.

**Comment by danielr1996** [comment]
https://github.com/danielr1996/kubectl-action

**Reply by kshitij-halankar** [reply]
@danielr1996 done!

**Reply by gogson** [reply]
@danielr1996 done ! https://github.com/gogson/MidjourneyAssistant

**Reply by WHAHA-HA** [reply]
@danielr1996, @gogson
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by WHAHA-HA** [comment]
@kshitij-halankar 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by kshitij-halankar** [reply]
@WHAHA-HA done

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by ghost** [comment]


**Reply by gogson** [reply]
@AlekseevDanil done ! https://github.com/gogson/MidjourneyAssistant

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ghost** [reply]
Done

https://github.com/Mr-Cuda/Fake-BTC

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by number0x0001** [comment]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by kshitij-halankar** [reply]
@number0x0001 done https://github.com/kshitij-halankar/TravelR

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by montasim** [reply]
Done @ArielMAJ

**Reply by ArielMAJ** [reply]
Ty

**Comment by HHNebula** [comment]
Done. https://github.com/HHNebula/HHNebula

**Reply by gogson** [reply]
@HHNebula done!

**Reply by Buedenbender** [reply]
@HHNebula Done 
https://github.com/Buedenbender/datscience

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by 41pha1** [comment]
Done
https://github.com/41pha1/MIDI-Converter

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ghost** [reply]
Done

https://github.com/Mr-Cuda/Fake-BTC

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by aarongeo1** [comment]
done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ghost** [reply]
Done 

https://github.com/Mr-Cuda/Fake-BTC

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by Buedenbender** [comment]


**Reply by ghost** [reply]
Done 

https://github.com/Mr-Cuda/Fake-BTC

**Reply by ghost** [reply]
Ty :)

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by ghost** [comment]
Done

https://github.com/Mr-Cuda/Fake-BTC

**Reply by montasim** [reply]
Please do it for me.
https://github.com/montasim/montasim

**Reply by ghost** [reply]
Ok, can you star my repo?
https://github.com/Mr-Cuda/Fake-BTC

After that i'll star urs

**Reply by montasim** [reply]
Done.

**Reply by ghost** [reply]
Ty :) 
Done

**Reply by montasim** [reply]
Done @Mr-Cuda

**Reply by ghost** [reply]
I know i already starred ur repo

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by kensoz** [comment]


**Reply by montasim** [reply]
Star my repo first. Then I will give you star.
https://github.com/montasim/montasim

**Reply by kensoz** [reply]
@montasim 
Done https://github.com/kensoz/FEC

**Reply by montasim** [reply]
Done @kensoz

**Reply by ghost** [reply]
@kensoz 

https://github.com/Mr-Cuda/Fake-BTC

Can you do the game for me?

**Reply by kensoz** [reply]
@Mr-Cuda 
Done https://github.com/kensoz/FEC

**Reply by ghost** [reply]
Ty
Done

**Reply by bensonarafat** [reply]
Done https://github.com/codepymdev/grad

**Comment by ArielMAJ** [comment]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@kshitij-halankar I still haven't received your star, so I'll be removing yours. Lmk when you star me so I'll star you back again!

**Comment by LauraBigoni** [comment]
Please help me star my repo: https://github.com/LauraBigoni/WeatherApp
I will star you back, just leave the link to your repo too. Thank you guys

**Reply by bensonarafat** [reply]
done https://github.com/codepymdev/grad

**Comment by bensonarafat** [comment]
https://github.com/codepymdev/grad

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Comment by DevElkami** [comment]
Hi everyone,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption (MAUI and Winform).

I would appreciate it if you could check it out and give it a star. Thank you.

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)
